testutil/lowlevel.go

```diff
@@ -414,3 +418,8 @@ func (sys *SyscallRecorder) Remove(name string) error {
 	call := fmt.Sprintf("remove %q", name)
 	return sys.call(call)
 }
+
+func (sys *SyscallRecorder) Fchdir(fd int) error {
```

Comment:
Nice, I'd only add the check that fd is an open fd like we do in other places in this module.